### PR TITLE
openshift components playbook

### PR DESCRIPTION
**Pull request by JayKayy**
#### What does this PR do?

Adds a playbook to tie in optional metrics and logging OSE components.(Other roles can be added in the future). This also adds all the necessary varibales for logging and metrics to the ose-provision inventory file
#### How should this be manually tested?

This is simply a 5-line playbook that conditionally ties in two already tested roles from openshift-ansible repo
#### Is there a relevant Issue open for this?

N/A
#### Who would you like to review this?

/cc @etsauer  @oybed @sabre1041

### CODE REVIEW

File: rhc-ose-ansible/inventory/ose-provision [hunk 1]
```diff
@@ -12,6 +12,33 @@ openstack_key_name=""
 #openshift_openstack_master_storage_size: 10
 #openshift_openstack_node_storage_size: 10
 
+#env_id=
+rhc_ose_inv_dest=.
+rhc_ose_cluster_method=native
```
Comment (etsauer) [inline rhc-ose-ansible/inventory/ose-provision:22]: why is this part of this playbook? seems unrelated.

File: rhc-ose-ansible/inventory/ose-provision [hunk 2]
```diff
@@ -12,6 +12,33 @@ openstack_key_name=""
...
+rhc_ose_cluster_public_hostname=public.hostname.com
+rhc_ose_cluster_hostname=hostname.com
+#rhc_ose_ssh_user=root
+#rhc_ose_identity_provider=htpasswd
+deployment_type=openshift-enterprise
+
+#Openshift Components
+rhc_ose_enable_efk=true                  #If E-F-K logging stack should be deployed
```
Comment (etsauer) [inline rhc-ose-ansible/inventory/ose-provision:57]: can we just use the standard vars from openshift-ansible for this? i.e. https://github.com/openshift/openshift-ansible/tree/master/playbooks/adhoc/metrics_setup

File: rhc-ose-ansible/inventory/ose-provision (hunk 1 above)
Comment (JayKayy) [inline rhc-ose-ansible/inventory/ose-provision:22]: These are additions from the openshift-install PR. I used the same inventory file from the openshift-install role, since you would be using the same inventory file, as this would all be a part of the same play?

File: rhc-ose-ansible/inventory/ose-provision (hunk 2 above)
Comment (JayKayy) [inline rhc-ose-ansible/inventory/ose-provision:57]: I had set this up for some consistency. for example if just using their vars we would have:

``` ...
rhc_ose_enable_efk=true 
openshift_hosted_metrics_deploy=true
...
```

(my efk doesnt have a `deploy=true` variable equivalent just a cleanup variable)

 Just didnt think it looks very nice and also a bit misleading. But that was the reasoning for that. I could change it back which wouldn't be a problem.

Also i am not using that metrics playbook but the openshift_metrics role: https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_metrics.

Maybe openshift_ansible has some redundancies? I think we are better off sticking to the roles as opposed to that adhoc playbook. Let me know what you think.

**Comment by etsauer** [comment]
@JayKayy could you update this from the latest upstream so we can explore this some more?

**Comment by JayKayy** [comment]
@etsauer updated
